### DISCUSSION TITLE
Trying catching up... the release is ultra fast!!! OMG

### DISCUSSION THREAD

**Discussion by ingted** [NTDLS/Katzebase#93] [General]
Katzebase is ultra cool!! Can't agree anymore... right!!??

**Comment by NTDLS** [comment]
I’ll slow down soon. Been putting in a lot of hours lately. Things are improving quickly!! 😅

**Reply by NTDLS** [reply]
I’m actually trying to focus on the UI now, so I’m just bug fixing the engine as I find stuff. I have no plans on changing anything major going forward. The next engine change will be LEFT OUTER, but I don’t think that’s going to be a big change at all given the new schema intersection methods.
